pkg/detectors/docusign/docusign.go

```diff
@@ -83,25 +81,14 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 					return nil, errors.WrapPrefix(err, "Error making request", 0)
 				}
 
-				// Read the response body
-				body, err := io.ReadAll(res.Body)
+				verifiedBodyResponse, err := common.ParseResponseForKeywords(res.Body, []string{"ey"})
```

Comment:
suggestion: Linter noticed the response body is not closed, looks like it got removed in the refactor. We should throw that back in here to prevent any resource leaks.